### PR TITLE
amendments to daml finance settlement docs

### PR DESCRIPTION
**Pull request by johan-da**

### CODE REVIEW

File: docs/2.8.0/docs/daml-finance/tutorials/advanced-topics/lifecycling/intermediated-lifecycling.rst [hunk 1]
```diff
@@ -175,12 +175,12 @@ What if one party wants to cancel the settlement?
...
 :ref:`Batch <type-daml-finance-interface-settlement-batch-batch-97497>` to cancel all associated
 :ref:`Instructions <type-daml-finance-interface-settlement-instruction-instruction-30569>`
 atomically.
 
-Parties who are not a requestor can prevent settlement by not approving / allocating their
-instructions (since a batch is only successful if the settlement instructions are fully allocated
-and approved).
+Settlement can be obstructed if the responsible parties do not carry out their duties to allocate
+and approve the settlement instructions, as the success of a batch hinges on these actions being
+completed.
```
Comment (marciw-da) [inline docs/2.8.0/docs/daml-finance/tutorials/advanced-topics/lifecycling/intermediated-lifecycling.rst:186]:
```suggestion
Settlement can also be obstructed if the responsible parties do not fully allocate
and approve the settlement instructions.

```

Comment (marciw-da) [reply docs/2.8.0/docs/daml-finance/tutorials/advanced-topics/lifecycling/intermediated-lifecycling.rst:186]: Note: I don't think the line about the "success of a batch" adds anything, so I deleted it. Feel free to veto that change and I'll try again.

And I added "also" to provide a connection between the two paragraphs, which was a little clearer before these edits (though I see why the revisions are needed).

**Review by marciw-da — COMMENTED** [review]
One suggestion

**Review by marciw-da — APPROVED** [review]
LGTM. Thanks!

**Review by matteolimberto-da — APPROVED** [review]

**Pull request merged**